src/autolabel/labeler.py

```diff
@@ -168,40 +157,7 @@ def run(
 
         dataset = dataset.get_slice(max_items=max_items, start_index=start_index)
 
-        if self.create_task:
-            self.db.initialize()
-            self.dataset_obj = self.db.initialize_dataset(dataset.df, self.config)
-            self.task_object = self.db.initialize_task(self.config)
-        else:
-            self.all_annotations = []
-
-        if isinstance(dataset, str):
-            csv_file_name = (
-                output_name
-                if output_name
-                else f"{dataset.replace('.csv','')}_labeled.csv"
-            )
-        else:
-            csv_file_name = f"{self.config.task_name()}_labeled.csv"
-
-        if self.create_task:
-            # Initialize task run and check if it already exists
-            self.task_run = self.db.get_task_run(
-                self.task_object.id, self.dataset_obj.id
-            )
-            # Resume/Delete the task if it already exists or create a new task run
-            if self.task_run:
-                logger.info("Task run already exists.")
-                self.task_run = self.handle_existing_task_run(
-                    self.task_run,
-                    csv_file_name,
-                    gt_labels=dataset.gt_labels,
-                    additional_metrics=additional_metrics,
-                )
-            else:
-                self.task_run = self.db.create_task_run(
-                    csv_file_name, self.task_object.id, self.dataset_obj.id
-                )
+        self.all_annotations = []
```

Comment:
this shouldnt be an object property, lets keep the scope to this method only

Reply:
makes sense!